src/app/pages/sign-out-confirm/sign-out-confirm-layout.tsx

```diff
@@ -28,11 +28,20 @@ export const SignOutConfirmLayout: FC<SignOutConfirmLayoutProps> = props => {
       <Box mx="loose" mb="extra-loose">
         <form onChange={form.handleChange} onSubmit={form.handleSubmit}>
           <Body>
-            When you sign out,
-            {whenWallet({
-              software: ` you'll need your Secret Key to sign back in. Only sign out if you've backed up your Secret Key.`,
-              ledger: ` you'll need to reconnect your Ledger to sign back into your wallet.`,
-            })}
+            <p>
+              When you sign out,
+              {whenWallet({
+                software: ` you'll need your Secret Key to sign back in. Only sign out if you've backed up your Secret Key.`,
+                ledger: ` you'll need to reconnect your Ledger to sign back into your wallet.`,
+              })}
+            </p>
+            <p style={{ marginTop: '24px', fontWeight: 'bold' }}>
+              {whenWallet({
+                software:
+                  "⚠️ If you haven't backed up your Secret Key, you will loose all your funds.",
+                ledger: '',
```

Comment:
I'm not sure how it works with Ledger, do we need a message here to?

Reply:
I don't believe I more strong message is needed. The key is on their Ledger, they can always sign back in with it.